### PR TITLE
refactor(cli,migrate): rename introspect to db pull

### LINKED ISSUE #5836: Add `prisma db pull` command for introspection
* Add `prisma db pull` command for introspection, with the same behavior
* Document `prisma db pull` in CLI help output
* Should `prisma db pull` be added in Preview? If not, we should change the help output to position the individual commands as Preview rather than the namespace.

```
> npx prisma db -h
Environment variables loaded from prisma/.env

🏋️  Manage your database schema and lifecycle during development.

WARNING Prisma db is currently in Preview (https://pris.ly/d/preview).
There may be bugs and it's not recommended to use it in production environments.
When using any of the subcommands below you need to explicitly opt-in via the --preview-feature flag.

Flag

  --preview-feature   Run preview Prisma commands

Usage

  $ prisma db [command] [options] --preview-feature

Options

  -h, --help   Display this help message
    --schema   Custom path to your Prisma schema

Commands

        push   Push the state from Prisma schema to the database during prototyping (preview)
        pull   Pull the state from the database to the Prisma schema using introspection
        seed   Seed your database (preview)

Examples

  Using prisma db push
  $ prisma db push --preview-feature

  Using prisma db seed
  $ prisma db seed --preview-feature
```

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/prisma/issues/5836

This renames `introspect` to `db pull`

We want to keep both without a deprecation warning.

### CODE REVIEW

File: src/packages/migrate/src/__tests__/DbPull.test.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import path from 'path'
-import { Introspect } from '../Introspect'
+import { DbPull } from '../commands/DbPull'
 import { consoleContext, Context } from './__helpers__/context'
-import {
-  SetupParams,
-  setupPostgres,
-  tearDownPostgres,
-} from '../utils/setupPostgres'
+// import
```
Comment (williamluke4) [inline src/packages/migrate/src/__tests__/DbPull.test.ts:3]: Is this intended

Comment (Jolg42) [reply src/packages/migrate/src/__tests__/DbPull.test.ts:3]: It's not used, doesn't change anything, seems only the code using it was commented (a long time ago?)

**Review by timsuchanek — APPROVED** [review]
LGTM

File: src/packages/cli/src/CLI.ts [hunk 2]
```diff
@@ -144,6 +140,9 @@ export class CLI implements Command {
       Create migrations from your Prisma schema, apply them to the database, generate artifacts (e.g. Prisma Client)
       ${chalk.dim('$')} prisma migrate dev --preview-feature
   
+      Introspect an existing database
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply src/packages/cli/src/CLI.ts:143]: Done

**Pull request merged**